### DISCUSSION TITLE
This tool is fantastic

### DISCUSSION THREAD

**Discussion by hempels** [audiamus/AaxAudioConverter#49] [General]
Thank you!

**Comment by DistractedCanuck** [comment]
Just wanted to add that it is indeed Fantastic! Thank You!

**Comment by ghost** [comment]
I also would like to thank the author very much! So awesome and easy to use. It took me a bit to find this great tool because you have to dive deep into forums :-)

**Comment by GallupinGallup** [comment]
This is awesome! Thank you! It took me nearly 2.5 hours to find this but it worked perfectly!

**Comment by CedricBergande** [comment]
I was pretty annoyed with how aweful the audible app has become... then I found this app! Thank you sooo much for all your effort you put into this application, it really is fantastic :)
